packages/cosmos-client/src/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { RequestResponse } from "@dcos/http-service";
+
+export function buildRequestHeader(
+  action: string,
+  actionType: string,
+  entity: string,
+  version: string
+) {
+  return `application/vnd.dcos.${entity}.${action}-${actionType}+json;charset=utf-8;version=${version}`;
+}
+
+export function getErrorMessage(reqResp: RequestResponse<any>): string {
```

Comment:
Let's use the full form `reqResp` -> `requestResponse` or even destruct the argument right there into `{ response, message }`.

Reply:
`RequestResponse<any>` can we be more specific and avoid `any` here?

Reply:
We currently seem to have a working solution, that parses out error messages for all kinds of responses - those being rightfully represented by the `any` IMHO.

Modelling types in order to not have an any here actually would have quite some impact, given that we

1. do not seem have a [properly modelled](https://package.elm-lang.org/packages/elm/http/latest/Http#Error) errors for `RequestResponse` yet.
2. seem to have no documentation about the errors that might be returned by the Cosmos-API.
3. seem to not have an established pattern for how we want to model error-streams with RxJS yet, if i groked the discussions regarding `partition` and alike correctly.

Being an advocate of elm, where everything is typed in a sound way, I'd still propose to just go with the any for now.

Reply:
Addressed the first comment. If there are no objections we'll leave the any here for now.